src/chains/ethereum/ethereum/tests/miner/miner.test.ts

```diff
@@ -38,10 +38,11 @@ describe("miner", async () => {
         miner: {
           blockGasLimit: "0x5208" // 21000, or sending one empty tx
         },
-        chain: { chainId: 1337 }
+        chain: { chainId: 1337 },
+        logging: { logger: console }
```

Comment:
I think `EthereumOptionsConfig.normalize` will set this to console.log automatically. Was that not happening here? :thinking:

Reply:
Nope, my mistake from when I was mass updating to match the new constructor